.github/workflows/main.yaml

```diff
@@ -392,6 +392,21 @@ jobs:
           path: |
             /tmp/cloudcore.log
             /tmp/edgecore.log
+  check-api-docs:
```

Comment:
I have a new idea:  we don't need add a new job here. We can add a verify under `make verify`  in https://github.com/kubeedge/kubeedge/blob/master/Makefile#L67.  This job https://github.com/kubeedge/kubeedge/blob/master/.github/workflows/main.yaml#L25 will run `make verify`.